test/functional/multiwallet.py

```diff
@@ -15,30 +15,41 @@ class MultiWalletTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
-        self.extra_args = [['-wallet=w1', '-wallet=w2', '-wallet=w3']]
+        self.extra_args = [['-wallet=w1.dat', '-wallet=w2.dat', '-wallet=w3.dat']]
```

Comment:
Wallet names are currently allowed to not end with `.dat`. This test verified that we don't regress that support.

I'm not sure if we should go on supporting that, but we should discuss that before removing support.

Reply:
Agreed, I'm unsure if there are any real benefits of adding an `available` list to the listwallets RPC after all, or if it just complicates the PR. Will discuss on IRC and then remove if not.